docs/Testing.md

```diff
@@ -0,0 +1,69 @@
+# Testing
+
+Moya has been created with testing at its heart. In this document, you will find all the customization points that will allow you to tailor your testing.
+
+## `sampleData`
+
+When creating your `TargetType` you have been required to provide `sampleData` for your targets. As you already know, all you want to do there is to provide `Data` that represents a sample response from every particular target.
```

Comment:
I think we should avoid making assumptions about user knowledge here. Using `As you already know` can cause some bad feelings to users who are not aware of what `sampleData` does/means.

More in [this post](http://alistapart.com/blog/post/the-most-dangerous-word-in-software-development)